composer.json

```diff
@@ -29,7 +29,7 @@
     },
     "require-dev": {
         "friendsofphp/php-cs-fixer": "~2.16.1",
-        "phpstan/phpstan": "^0.12",
+        "phpstan/phpstan": "^0.11 || ^0.12",
```

Comment:
Why do we need 0.11?

Reply:
Trying to get PHP 7.1 to pass - if we put `phpstan` into `composer.json` then for the PHP 7.1 builds we have to make a `phpstan` available that supports PHP 7.1
`composer` failed https://travis-ci.com/sabre-io/skel/jobs/282746328
`Your requirements could not be resolved to an installable set of packages.`
But `phpstan` still officially supports PHP 7.1
Hmmm - I think I didn't look properly at the next Travis build. I don;t think I need this `0.11` at all!

Reply:
I have removed this change. It was not needed.